src/components/MyAccount/CheckoutsTab/CheckoutsTab.test.tsx

```diff
@@ -33,11 +33,14 @@ describe("CheckoutsTab", () => {
     const component = render(
       <CheckoutsTab patron={processedPatron} checkouts={processedCheckouts} />
     )
+    const renewableCheckout = processedCheckouts[0]
+    const row = component.getByText(renewableCheckout.title).closest("tr")
```

Comment:
I just noticed this does `component.getBy...` where `component` is coming from the render function. Usually, it's  `screen.getBy...` and screen is imported from the package. Did the latter not work and you had to use `component` from the render function? They're essentially the same so more out of curiosity.

Reply:
No this was inadvertent. I'll update.

Reply:
Ok, actually... Component is returned by render which comes from our wrapped methods. I'm not exactly sure why that's the pattern here, but it's working, and I'm not so inclined to fix it. Unless you think its important to have it standardized.

Reply:
ah, right. it's ok to keep